### PR TITLE
Libvpx: bump version

### PR DESCRIPTION
**Pull request by kenmays**
Closes https://github.com/haikuports/haikuports/issues/5070
All unit tests passed.

### CODE REVIEW

**Comment by waddlesplash** [comment]
This includes a SONAME change.

File: media-libs/libvpx/libvpx-1.8.2.recipe [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+libVersion="5.0.0"
```
Comment (Begasus) [inline media-libs/libvpx/libvpx-1.8.2.recipe:31]: Version change here:
```diff
- libVersion="5.0.0"
+ libVersion="6.2.0"
```

File: media-libs/libvpx/libvpx-1.8.2.recipe [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+COPYRIGHT="2009-2012 Google Inc.,
+	The Mozilla Foundation and
+	The Xiph.Org Foundation"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/webmproject/libvpx/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="8735d9fcd1a781ae6917f28f239a8aa358ce4864ba113ea18af4bb2dc8b474ac"
+SOURCE_FILENAME="libvpx-$portVersion.tar.gz"
+SOURCE_DIR="libvpx-$portVersion"
+#PATCHES="libvpx-$portVersion.patchset"
```
Comment (Begasus) [inline media-libs/libvpx/libvpx-1.8.2.recipe:19]: It builds without the patchset, if it's really not needed this line could be removed

**Comment by Begasus** [comment]
Packages deppending on libvpx should need a revbump after this is merged (and some changes, some require >= 5 atm)

File: media-libs/libvpx/libvpx-1.8.2.recipe (hunk 1 above)
Comment (kenmays) [reply media-libs/libvpx/libvpx-1.8.2.recipe]: fixed

File: media-libs/libvpx/libvpx-1.8.2.recipe (hunk 2 above)
Comment (kenmays) [reply media-libs/libvpx/libvpx-1.8.2.recipe]: fixed

**Comment by Begasus** [comment]
Thanks! Could you list up depending packages for this one and do a revbump on those after this is merged?

**Comment by kenmays** [comment]
Sure.

**Pull request merged**